### PR TITLE
Add copyright headers and contributing guidelines.

### LINKED ISSUE #169: Source files are missing copyright headers
All the dart source files in the project should have a standard copyright header.

### PR DESCRIPTION
**Pull request by oSoMoN**
Fixes #169, #170.

### CODE REVIEW

**Review by Feichtmeier — APPROVED** [review]
LGTM - everything as discussed :)

@madsrh @Jupi007 any objections?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+## Code of Conduct
+
+This project is subject to the [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct) to foster an open and welcoming place to contribute.
+By participating in the project (in the form of code contributions, issues, comments, and other activities), you agree to abide by its terms.
+
+## Pull requests
+
+Changes to this project should be proposed as pull requests on GitHub at: <https://github.com/ubuntu-flutter-community/software>.
```
Comment (Jupi007) [inline CONTRIBUTING.md:10]: Maybe specify that you need a linked issue to publish a PR?
So we can avoid spam notifications about CI.

**Comment by Jupi007** [comment]
I added a comment :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (Feichtmeier) [reply CONTRIBUTING.md:10]: Good idea

Comment (oSoMoN) [reply CONTRIBUTING.md]: Very good idea, done!

**Review by Jupi007 — APPROVED** [review]
LGFM :+1:

**Review by madsrh — APPROVED** [review]

**Review by Feichtmeier — APPROVED** [review]

**Pull request merged**